src/utils/derive_ipfs_cid.test.ts

```diff
@@ -0,0 +1,17 @@
+import { expect } from 'chai';
+import { deriveIpfsCid } from './derive_ipfs_cid';
+
+const dataToHash = [
+	{ data: 'hello', hash: 'QmWfVY9y3xjsixTgbd9AorQxH7VtMpzfx2HaWtsoUYecaX' },
+	{ data: 'my file data', hash: 'Qmd8s28wFUPP2rRbQEvvoHh84G22ShP5YzNgvT2mdhyrmq' },
+	{ data: Buffer.from('my file data'), hash: 'Qmd8s28wFUPP2rRbQEvvoHh84G22ShP5YzNgvT2mdhyrmq' }
+];
+
+describe('deriveIpfsCid function', () => {
+	it('returns the expeced hash', async () => {
+		for (const expected of dataToHash) {
```

Comment:
Consider the variable name `testCaseData` over `expected`.

Reply:
8d348aa8aeb59d755388ae72a0415036253cfcce